.travis/deploy_sbt.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o verbose
+
+# If we're on the master branch and in an application project, we should
+# build a new Docker image and push it to ECR.
+
+if [[ "$BRANCH" != "master" ]]
+then
+  echo "Not on master (BRANCH=$BRANCH); skipping deploy..."
+  exit 0
+fi
+
+if [[ "$PROJECT" == "common" ]]
+then
+  echo "Common lib doesn't have a Docker container; skipping deploy..."
+  exit 0
+fi
```

Comment:
👍 i'm not sure it should be in the `run_script.sh` but it's appealing to keep the logic out of this script.